README.rst

```diff
@@ -25,7 +25,7 @@ The project uses buildout to bootstrap::
 
     python bootstrap.py
 
-    bin/buildout -N
+    python bin/buildout -N
```

Comment:
@chaudum true, but I figure it makes the instructions simpler to list one command that works for everyone?